src/build.ts

```diff
@@ -54,36 +77,32 @@ const createFunctionFile = async (
   const fnFileRelative = path.join("functions", `${fnId}.js`);
   const fnBundledPath = path.join(outputDirectory, fnFileRelative);
 
-  // We'll default to just using whatever Deno executable is on the path
-  // Ideally we should be able to rely on Deno.execPath() so we make sure to bundle with the same version of Deno
-  // that called this script. This is perhaps a bit overly cautious, so we can look to remove the defaulting here in the future.
-  let denoExecutablePath = "deno";
   try {
-    denoExecutablePath = Deno.execPath();
-  } catch (e) {
-    protocol.error("Error calling Deno.execPath()", e);
-  }
-
-  try {
-    // call out to deno to handle bundling
-    const p = Deno.run({
-      cmd: [
-        denoExecutablePath,
-        "bundle",
-        "--quiet",
-        fnFilePath,
-        fnBundledPath,
-      ],
+    await DenoBundler.bundle({
+      entrypoint: fnFilePath,
+      outFile: fnBundledPath,
     });
+  } catch (denoBundleErr) {
+    if (!(denoBundleErr instanceof BundleError)) {
+      protocol.error(`Error bundling function file "${fnId}" with Deno`);
+      throw denoBundleErr;
+    }
+
+    protocol.warn(
```

Comment:
I put this in, but after further testing I don't think the user needs to be informed of this, lmk what you think

Reply:
Makes sense 👍  I think that would require both an evolution of the deno-slack-protocols library as well as the slack-cli.